notebooks/06-cifar10-baseline.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p><code>self.log('train_loss', loss)</code></p><p>just to match it with other keys.</p><p>---</p><p> </p><p>mind clean configure_optimizer a bit? since notebook render on github is not so good.</p><pre class="ql-syntax" spellcheck="false"># add in imports
from torch.optim.lr_scheduler import OneCycleLR

optimizer = ...
steps_per_epoch = 

lr_dict = {
    'scheduler': OneCycleLR(optimizer, 0.1, epoch=..., steps_per_epoch=...),
    'interval': 'step',
}

return {
    'optimizer': optimizer,
    'lr_scheduler': lr_dict,
}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/pytorch-lightning/pull/4818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>